src/ussd_higherhealth_healthcheck.js

```diff
@@ -224,6 +225,714 @@ go.app = (function () {
+                "Main Campus"
+        ],
+            "Sekhukhune":[
+                "Apel",
+                "CN Phatudi",
+                "CS Barlow",
+                "Central Office"
+        ],
+            "Sol Plaatje University (SPU)":[
+                "Kimberley"
+        ],
+            "South Cape":[
+                "Beaufort West",
+                "Bitou",
+                "Central Office",
+                "George",
+                "Hessequa",
+                "Mossel Bay",
+                "Oudtshoorn"
+        ],
+            "South West Gauteng ":[
+                "Dobsonville",
+                "George Tabor",
+                "Land is Wealth Farm",
+                "Molapo",
+                "Roodepoort",
+                "Roodepoort West",
+                "Technisa"
+        ],
+            "Stellenbosch University (SU)":[
+                "Stellenbosch",
+                "Tygerberg"
+        ],
+            "Taletso":[
+                "Central Office",
+                "Lehurutshe",
+                "Lichtenburg",
+                "Mafikeng"
+        ],
+            "Thekwini":[
+                "Asherville",
+                "Cato Manor",
+                "Centec",
+                "Central Office",
+                "Melbourne",
+                "Springfield",
+                "Umbilo"
+
+        ],
+            "Tshwane North":[
+                "Central Office",
+                "Mamelodi",
+                "Pretoria",
+                "Rosslyn",
+                "Soshanguve North",
+                "Soshanguve South",
+                "Temba"
+
+        ],
+            "Tshwane South":[
+                "Atteridgeville",
+                "Central Office",
+                "Centurion",
+                "Odi",
+                "Pretoria West"
+
+        ],
+            "Tshwane University of Technology (TUT)":[
+                "Polokwane",
+                "Mbombela",
+                "eMalahleni",
+                "Arcadia",
+                "Arts",
+                "Ga-Rankuwa",
+                "Pretoria West",
+                "Soshanguve"
+        ],
+            "UNISA":[
+                "Bloemfontein",
+                "Kroonstad",
+                "Durban",
+                "Newcastle",
+                "Pietermaritzburg",
+                "Richards Bay",
+                "Wild Coast (Mbizana)",
+                "East London",
+                "Mthatha",
+                "Port Elizabeth",
+                "Ekurhuleni",
+                "Florida (Science Campus)",
+                "Johannesburg",
+                "Pretoria (Sunnyside)",
+                "Vaal",
+                "George",
+                "Parow",
+                "Giyani",
+                "Makhado",
+                "Polokwane",
+                "Kimberley",
+                "Mafikeng",
+                "Potchefstroom",
+                "Rustenburg",
+                "Mbombela",
+                "Middelburg"
```

Comment:
I don't think that this data structure will work.

An institution like UNISA has campuses all over the country. If the user selected "Western Cape", then we only want to display the western cape campuses, not all the campuses across the country. This list is way too big to be going through page-by-page on USSD.

I would suggest something like:
```
{
  "ZA-WC": {
    "UNISA": [
      "George",
      "Parow"
    ],
    ...
  },
  ...
}